src/gtk/button.cpp

```diff
@@ -132,7 +132,7 @@ bool wxButton::Create(wxWindow *parent,
     wxGCC_WARNING_SUPPRESS(deprecated-declarations)
     gtk_button_set_alignment(GTK_BUTTON(m_widget), x_alignment, y_alignment);
     wxGCC_WARNING_RESTORE()
-#endif
```

Comment:
Add `#endif` hint.

Reply:
PTAL fixed in [d34fe0f](https://github.com/artur-so/wxWidgets/pull/1/commits/d34fe0f1e680a6cb06cdddd6a956fc1ba6d16f6b)